config/deploy.rb

```diff
@@ -229,8 +229,11 @@
       abort "Exiting because you didn't type 'yes'" unless STDIN.gets.chomp == 'yes'
     end
 
-    backup.db.dump
-    backup.db.trim
+    print "Backup database? [no/YES] ".colorize(:green)
+    unless STDIN.gets.chomp.downcase == 'no'
```

Comment:
I'm not sure if this is intended, but this would backup the db if anything except "no" if entered (not just "yes").